### DISCUSSION TITLE
What is the difference between ensemble tiling and standard tiling

### DISCUSSION THREAD

**Discussion by JinYuannn** [open-edge-platform/anomalib#2299] [Q&A] [answered]
The paper mentions that each tiler location uses a different model.
![image](https://github.com/user-attachments/assets/b9c254ab-48af-4249-9336-a95c6143d0ac)

But in the code, the image is simply divided into patches to form a larger batch of data, which is then fed into the same feature extractor.
https://github.com/openvinotoolkit/anomalib/blob/cdd338c8992efab6032e7e29d0069ec3f13d5bbb/src/anomalib/models/image/reverse_distillation/torch_model.py#L69C9-L71C48

However, a single model on the combined set of all tile locations could lead to a loss of positional information, with a potential negative effect on the performance of models that perform better on aligned objects.
-- from the paper

Is it an issue with the code implementation, or is it a problem with my understanding?

**Comment by samet-akcay** [comment, marked as answer]
@JinYuannn the code you are referring to is not the ensemble tiling approach. Please refer to this PR 
https://github.com/openvinotoolkit/anomalib/pull/1226

> NOTE
This PR is still open and has not been merged. @blaz-r is working on it to get that merged.
